src/editor/PopupPreview.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * WordPress dependencies
+ */
+import { Button } from '@wordpress/components';
+import { withSelect, withDispatch } from '@wordpress/data';
+import { compose } from '@wordpress/compose';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * External dependencies
+ */
+import { stringify } from 'qs';
+import { WebPreview } from 'newspack-components';
+
+const PopupPreviewSetting = ( { savePost, isSavingPost, postId, metaFields } ) => {
+	const query = stringify( {
+		newspack_popups_preview_id: postId,
+		// Autosave does not handle meta fields, so these will be passed in the URL
+		...metaFields,
+	} );
+
+	return (
+		<WebPreview
+			url={ `/?${ query }` }
+			renderButton={ ( { showPreview } ) => (
+				<Button
+					isLink
```

Comment:
When the autosave is in progress this button gets the in-progress state of the Update button, which looks peculiar on a link. I'd suggest changing it to `isPrimary` as discussed in https://github.com/Automattic/newspack-popups/pull/36#issuecomment-579145740. Is it possible to place it below `Move to Trash` as Thomas suggested or are we locked in to the current position?

Reply:
That's true that the in-progress is odd… 

![Screen Capture on 2020-01-30 at 16-42-28](https://user-images.githubusercontent.com/7383192/73464736-a9638280-437f-11ea-842a-52468ff1baec.gif)

How about keeping this as a link, but dimming on save? Otherwise, we can go back to using a `Button`. WDYT @thomasguillot ?

---

Different placement is not possible, we're locked in to this space:

<img width="286" alt="Edit Pop-up ‹ newspack test — WordPress 2020-01-30 16-40-28" src="https://user-images.githubusercontent.com/7383192/73464465-4671eb80-437f-11ea-8872-0f426b99e16e.png">

Reply:
Let's use a primary button.

It's ok if we can't move it underneath.

Reply:
Updated: 80235dc